Automattic/simplenote-ios
# Switch configuration to trunk

### Fix
This PR updates `.configure` to use trunk instead of main.
It also updates `fastlane` and `release-toolkit` to the latest version.

### Test
1. `bundle install`
2. Run `bundle exec fastlane run configure_apply`
3. Verify that the tool succeeds. If you already had the latest configuration, you shouldn't have any changes.
4. Delete all the files in the `files_to_copy` section of `.configure` from your project directory.
5. Run `bundle exec fastlane run configure_apply` again and verify that the file you deleted have been recreated.

### Review
Only one developer is required to review these changes, but anyone can perform the review.

### Release
 These changes do not require release notes.
